pkg/reconciler/brokercell/broker_targets_config.go

```diff
@@ -117,6 +118,11 @@ func (r *Reconciler) addToConfig(_ context.Context, b *brokerv1beta1.Broker, tri
 						Topic:        brokerresources.GenerateRetryTopicName(t),
 						Subscription: brokerresources.GenerateRetrySubscriptionName(t),
 					},
+					SubscriberInfo: &config.SubscriberInfo{
+						SubscriberType:      getSubscriberType(t.Kind, t.APIVersion),
+						SubscriberNamespace: t.Spec.Subscriber.Ref.Namespace,
```

Comment:
Just to be sure, what defaults a value into `t.Spec.Subscriber.Ref.Namespace`? Is it the webhook? Or the URL lookup code? If it is the code, then this might be the empty string and we would need to use `t.Namespace` when this is the empty string.